securedrop_client/storage.py

```diff
@@ -177,18 +203,17 @@ def update_submissions(remote_submissions, local_submissions, session, data_dir)
             # A new submission to be added to the database.
             _, source_uuid = submission.source_url.rsplit('/', 1)
             source = session.query(Source).filter_by(uuid=source_uuid)[0]
-            ns = Submission(source=source, uuid=submission.uuid,
-                            size=submission.size,
-                            filename=submission.filename,
-                            download_url=submission.download_url)
+            ns = model(source_id=source.id, uuid=submission.uuid, size=submission.size,
+                       filename=submission.filename, download_url=submission.download_url)
             session.add(ns)
             logger.debug('Added new submission {}'.format(submission.uuid))
 
     # The uuids remaining in local_uuids do not exist on the remote server, so
     # delete the related records.
     for deleted_submission in [s for s in local_submissions
                                if s.uuid in local_uuids]:
-        delete_single_submission_or_reply_on_disk(deleted_submission, data_dir)
+        if is_file:
```

Comment:
I was assuming we would only store the temp file used for decryption then load it into the database. This was kind of a leap since the full functionality of this is split across tickets.